docs/assets/js/sidebar.js

```diff
@@ -0,0 +1,27 @@
+function init() {
+  const secondaryNavCat = document.querySelector( '.ds-nav-container' );
+
+  /**
+   * Test the viewport size and set whether the test passes on the instance.
+   */
+  function handleViewportChange() {
+    if ( window.innerWidth < 601 ) {
+      secondaryNavCat.removeAttribute( 'open' );
```

Comment:
Adding/removing the native `open` attribute is real clever. 👍

Reply:
Thanks!